legendary/cli.py

```diff
@@ -2507,20 +2508,68 @@ def move(self, args):
         old_base, game_folder = os.path.split(igame.install_path.replace('\\', '/'))
         new_path = os.path.join(args.new_path, game_folder)
         logger.info(f'Moving "{game_folder}" from "{old_base}" to "{args.new_path}"')
-
         if not args.skip_move:
+            if not os.path.exists(args.new_path):
+                os.makedirs(args.new_path)
+            if os.stat(args.new_path).st_dev != os.stat(igame.install_path).st_dev:
+                choice = get_boolean_choice(f'Moving files to another drive can cause errors when running the game, '
+                                            f'however, it is unlikely to happen. Do you still wish to continue? (default=no)', default=False)
+                if not choice:
+                    logger.info(f'To move it without a risk move the folder manually to {new_path} '
+                                f'and run "legendary move {app_name} "{args.new_path}" --skip-move"')
```

Comment:
To be honest, I don't know. I thought that because this was not done before there must be a reason for it. On python docs I found that this function may or may not copy the metadata so I thought that must've been the reason but I've had 0 problems after trying it with 3-4 games so I'm sure it can be removed.